src/test/groovy/com/bugsnag/android/gradle/SmokeTest.groovy

```diff
@@ -0,0 +1,14 @@
+package com.bugsnag.android.gradle
+
+import org.junit.Assert
+import org.junit.Test
+
+
+class SmokeTest {
+
+    @Test
+    public void greeterPluginAddsGreetingTaskToProject() {
+        Assert.assertTrue(2 + 2 == 4)
```

Comment:
🤔 💭

Reply:
I've updated this to add a basic test of the plugin's functionality.